### PR TITLE
This will specify a readme for the repo

### PR DESCRIPTION
**Pull request by pvienneau**

### CODE REVIEW

**Review by devriestyler — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
 # react-style-manager
+
+[![CircleCI](https://circleci.com/gh/mb3online/react-style-manager.svg?style=shield&circle-token=91dc3f3ee64e7396e268b64d2bb70b243e79e73b)](https://circleci.com/gh/mb3online/react-style-manager)
+
+A react package to more easily manage the stateful styling of your components.
+
+## Getting started
+
+1. Run `npm install react-style-manager --save` to install the package in your application.
```
Comment (andrew09) [inline README.md]: Shouldn't this be yarn? and not --save?

**Review by andrew09 — CHANGES_REQUESTED** [review]
Looks good. Just 1 comment.

**Comment by andrew09** [comment]
@pvienneau can we rename this to be more in line with what it is doing?

File: README.md (hunk 1 above)
Comment (pvienneau) [reply README.md]: @andrew09 has been veto'd by group decision.

**Review by ericadamski — APPROVED** [review]

**Review by andrew09 — APPROVED** [review]
👍

**Pull request merged**